services/rfq/api/rest/server.go

```diff
@@ -258,6 +288,7 @@ func (r *QuoterAPIServer) AuthMiddleware() gin.HandlerFunc {
 				destChainIDs = append(destChainIDs, uint32(req.DestChainID))
 				loggedRequest = &req
 			}
+			useV1 = true
```

Comment:
Don't we need to derive the `useV1` flag from the request's origin/dest fast bridge addresses? Same applies to other cases